cmd/utils.go

```diff
@@ -1045,3 +1045,22 @@ func checkTime() {
 		}
 	}
 }
+
+func setNewIndexURL() {
+
+	var repoFile = getRepoFileLocation()
+	var oldIndexURL = "https://raw.githubusercontent.com/appsody/stacks/master/index.yaml"
+	var newIndexURL = "https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml"
+
+	data, err := ioutil.ReadFile(repoFile)
+	if err != nil {
+		Warning.log("Unable to read repository file")
+	}
+
+	replaceURL := bytes.Replace(data, []byte(oldIndexURL), []byte(newIndexURL), -1)
+
+	if err = ioutil.WriteFile(repoFile, replaceURL, 0666); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
I think we should remove the os.Exit
If this fails, the user may still be able to use appsody with the old index.

Reply:
yep this makes sense!